### ISSUE TITLE
More examples using Python API

### ISSUE THREAD

**Issue by coxley** [dropbox/pynsot#56]
Examples include using `Client` and `EmailHeaderClient`, but `get_api_client()` is the best way since it can auto-load from your `.pynsotrc` or be overridden.

In addition, examples showing every method being performed (post, patch, put, delete) would be convenient so you don't have to refer to the nsot web-ui's /api form.

**Comment by coxley** [comment]
As I'm going through the source to find the best way to utilize this for a project of mine, here are some things that need documented:

* `models.ApiModel`: Obviously the way to go for turning queries into python objects
* Documenting all the slumber functions that are hidden from dir() and help() on resource objects. These include methods like:

  * `get`
  * `post`
  * `patch`
  * `delete`
  * `query`

Basically should show how to do everything in the CLI examples via Python API.

**Comment by coxley** [comment]
Since this was last updated, there have been some improvements on the documentation side and ``models.ApiModel`` (which I had no idea I once discovered) has been replaced in favor of ``models.Resource`` derivatives.

Am submitting a PR that'll let this issue be closed, following then up on another to close #95

**Comment by jathanism** [comment]
Fixed in #99
